gg.minion.Minion.yml

```diff
@@ -10,10 +10,14 @@ finish-args:
   - --socket=x11
   - --share=network
   - --persist=.minion
+  # Steam-specific folders
   - --filesystem=~/.steam
-  - --filesystem=~/.local/share/Steam
+  - --filesystem=xdg-data/Steam
   - --filesystem=~/.var/app/com.valvesoftware.Steam
+  # Lutris installations
```

Comment:
Not a typo as lutris supports both wow and eso whereas epic doesn't support wow.